.github/workflows/test.yml

```
@@ -34,6 +34,8 @@ jobs:
         
         xcode: ["13.2.1", "12.5.1"]
 
+        a: ["a", "b", "c", "d"]
```

```suggestion
        # Run every combination of other matrix parameters four times to throw out flaky failures.
        a: ["a", "b", "c", "d"]
```
I assume this essentially forces four runs of every test to overcome flakiness... would you mind capturing this in a comment otherwise?

---

This is just a way for me to see if my fixes work. I'm not going to merge this.